src/ontology/make-bridge-ontologies-from-xrefs.pl

```diff
@@ -116,12 +116,14 @@
     print $fh "id: $prefix#develops_from\n";
     print $fh "equivalent_to: RO:0002225\n\n";
 
-    open(F,"ftr.obo");
```

Comment:
@cmungall please check

Reply:
```yaml
[Typedef]
id: foaf-depicted_by
name: depicted_by
is_metadata_tag: true
xref: http://xmlns.com/foaf/0.1/depicted_by

[Typedef]
id: foaf-homepage
name: homepage
is_metadata_tag: true
xref: http://xmlns.com/foaf/0.1/homepage

[Typedef]
id: dc-title
name: title
is_metadata_tag: true
xref: http://purl.org/dc/elements/1.1/title

[Typedef]
id: dc-contributor
name: contributor
is_metadata_tag: true
xref: http://purl.org/dc/elements/1.1/contributor

[Typedef]
id: dc-creator
name: creator
is_metadata_tag: true
xref: http://purl.org/dc/elements/1.1/creator

[Typedef]
id: dc-publisher
name: publisher
is_metadata_tag: true
xref: http://purl.org/dc/elements/1.1/publisher

[Typedef]
id: dc-description
name: description
is_metadata_tag: true
xref: http://purl.org/dc/elements/1.1/description

[Typedef]
id: dc-source
name: derived from resource
is_metadata_tag: true
xref: http://purl.org/dc/elements/1.1/source

[Typedef]
id: dcterms-isReferencedBy
name: is referenced by
is_metadata_tag: true
xref: http://purl.org/dc/terms/isReferencedBy

[Typedef]
id: dcterms-references
name: references
is_metadata_tag: true
xref: http://purl.org/dc/terms/references

[Typedef]
id: doap-GitRepository
name: Git repository
is_metadata_tag: true
xref: http://usefulinc.com/ns/doap#GitRepository

[Typedef]
id: doap-SVNRepository
name: SVN repository
is_metadata_tag: true
xref: http://usefulinc.com/ns/doap#SVNRepository

[Typedef]
id: doap-wiki
name: wiki
is_metadata_tag: true
xref: http://usefulinc.com/ns/doap#wiki

[Typedef]
id: doap-bug-database
name: bug database
is_metadata_tag: true
xref: http://usefulinc.com/ns/doap#bug-database

[Typedef]
id: seeAlso
name: see also
is_metadata_tag: true
xref: http://www.w3.org/2000/01/rdf-schema#seeAlso

[Typedef]
id: example_of_usage
name: example of usage
is_metadata_tag: true
xref: IAO:0000122
```

Reply:
@gouttegd can you add this file to the repo and double check that the resulting behaviour is still ok?

Reply:
If I add a `ftr.obo` file in `src/ontology/bridge` with the above contents, as expected I obtain the same OBO bridge files as before, with the added contents at the end of the file.

The `Typedef`s also end up as `owl:AnnotationProperty` stanzas in the corresponding OWL bridge files (which again is expected, since they are generated from the OBO bridge files).

Is that the intended result? Shall I commit that file?

Reply:
Just looking at a single bridge, are any of the properties used in ftr.obo _actually_ used in the bridge files?

Reply:
Some of them are used in the prologue, namely *seeAlso* and all the *dc-* properties (here with the Uberon/FBbt bridge):

```
ontology: uberon/bridge/uberon-bridge-to-fbbt
data-version: 2021-11-11
property_value: seeAlso "http://purl.obolibrary.org/obo/uberon/references/reference_0000026" xsd:anyURI
property_value: dc-title "Uberon bridge to fbbt" xsd:string
property_value: dc-description "Taxonomic equivalence axioms between the drosophila anatomy ontology and Uberon. Note that currently Uberon focuses on chordate anatomy so there are only a minimal set of equivalencies for high level classes. In future Uberon may bridge via the arthropod anatomy ontology" xsd:string
property_value: dc-creator "Uberon editors" xsd:string
property_value: dc-contributor "David Osumi-Sutherland" xsd:string
property_value: dcterms-references http://purl.obolibrary.org/obo/fbbt.owl
property_value: dcterms-references http://purl.obolibrary.org/obo/uberon.owl
```

Reply:
ok then, commit ftr.obo please and make sure its included! Thank you! :)

Reply:
Done, and the conflict introduced by the #2163 is fixed as well. Ready to merge everything if you’re happy.